pages/index.js

```diff
@@ -1,15 +1,25 @@
 import React from 'react'
 import { withRouter } from 'next/router'
 import { compose } from 'react-apollo'
+import { css } from 'glamor'
 
-import withAuthorization from '../components/Auth/withAuthorization'
+import { linkRule } from '@project-r/styleguide'
```

Comment:
linkRule is deprecated and will be removed once darkmode is fully implemented, use `import { A } from '@project-r/styleguide` instead